.github/ISSUE_TEMPLATE/critical_vulnerability_report.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Critical Vulnerability Report
+about: Critical vulnerability reports about the Raiden Network
+---
+
+# DO NOT use Github
+
+If the issue you want to report is a critical vulnerability, something that may lead to lock up or loss of funds for Raiden users, then please do not use the public issue tracker.
+
+Instead send an email with a detailed report to bounty@raiden.network.
+TODO: Also add a link to the bug bounty website rules here.
```

Comment:
Shouldn't we also tell the user to try reaching the Raiden team AFAP? Like personal chat on Gitter, or some other kind of communication platform?

Reply:
It should be our responsibility to ensure that emails sent to this address are read quickly. The more communication channels we advertise the bigger the chance that something is going to get lost.

Reply:
This email is the best way to reach us for security issues. It's sent to multiple people including myself. I scan it the minute I receive it.